sound/soc/sof/sof-priv.h

```diff
@@ -413,11 +413,13 @@ struct sof_ipc_fw_tracing_ops {
  * @ctx_save:		Optional function pointer for context save
  * @ctx_restore:	Optional function pointer for context restore
  * @set_core_state:	Optional function pointer for turning on/off a DSP core
+ * @set_pm_gate:	Function pointer for changing pm gate
```

Comment:
optional or mandatory?

Reply:
I thought we need to implement it but driver can work without it, so optional is better.